.golangci.yml

```diff
@@ -33,13 +33,19 @@ linters-settings:
   goimports:
     local-prefixes: github.com/golangci/golangci-lint
   gomnd:
-    # TODO(ldez) must be rewritten after the v1.44.0 release.
-    settings:
-      mnd:
-        # don't include the "operation" and "assign"
-        checks: argument,case,condition,return
-        ignored-numbers: 0,1,2,3
-        ignored-functions: strings.SplitN
+    # don't include the "operation" and "assign"
+    checks:
+      - argument
+      - case
+      - condition
+      - return
+    ignored-numbers:
+      - '0'
+      - '1'
+      - '2'
+      - '3'
```

Comment:
Must say a bit confusing that a list of numbers are treated as strings. (Don't mean this PR, saw this is how it's handled upstream but didn't know).

Reply:
Yes I know, I documented that https://golangci-lint.run/usage/linters/#gomnd.

Reply:
FYI it's for the compatibility with the old configuration.